game/views.py

```
@@ -494,6 +495,76 @@ def register_view(request):
+            for i in range(1, 5):
+                suggestion = f"{base_username}{i}"
+                if not User.objects.filter(username__iexact=suggestion).exists():
+                    suggestions.append(suggestion)
+                    if len(suggestions) >= 3:
+                        break
+        
+        # Add underscore suggestions if needed
+        if len(suggestions) < 3:
+            import random
+            while len(suggestions) < 3:
+                suffix = random.randint(10, 999)
```

Ruff (0.15.14)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)